airflow/providers/google/cloud/operators/gcs_to_gcs.py

```diff
@@ -277,14 +283,35 @@ def _copy_source_with_wildcard(self, hook, prefix):
                                      destination_object=destination_object)
 
     def _copy_single_object(self, hook, source_object, destination_object):
-        if self.last_modified_time is not None:
+        if self.last_modified_time and self.maximum_modified_time:
+            # check to see if object was modified between last_modified_time and
+            # maximum_modified_time
+            if hook.is_updated_between(self.source_bucket,
+                                       source_object,
+                                       self.last_modified_time,
+                                       self.maximum_modified_time
+                                       ):
+                self.log.debug("Object has been modified between %s and %s",
```

Comment:
I think you should rather log in debug when the object was excluded rather than included.

Reply:
Ok. I'll do that